### PR TITLE
Add PL-43499

### PR DESCRIPTION
**Pull request by brian-f-harness**
Add PL 1.20.9 early access feature: https://developer.harness.io/release-notes/platform#early-access-features

For reviewers, a preview is available: 

[Early access features](https://65a96053f6b16704b6606681--harness-developer.netlify.app/release-notes/early-access)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Comment by aimurphy** [comment]
Is this GA or Beta? I am getting conflicting info from here and this pr https://github.com/harness/developer-hub/pull/4971

File: release-notes/early-access.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ The following early access (beta) features are available for the Harness Platfor
 
 ### AIDA support automatically available
```
Comment (aimurphy) [inline release-notes/early-access.md:25]: haeding doesn't seem to match the feature flag description.

Comment (brian-f-harness) [reply release-notes/early-access.md:25]: Thanks @aimurphy

Comment (brian-f-harness) [inline release-notes/early-access.md:25]:
```suggestion
### Grant public access to Harness pipeline executions
```

**Comment by brian-f-harness** [comment]
> Are the 2 AIDA features (coveo and automatically available) in GA or Beta? I am getting conflicting info from here and this pr #4971

@aimurphy Nice catch. Yes, looks like we can move them to promoted to GA.

**Review by aimurphy — APPROVED** [review]

**Pull request merged**